src/coreclr/src/tools/Common/JitInterface/ThunkGenerator/ThunkInput.txt

```diff
@@ -199,6 +199,7 @@ FUNCTIONS
     CorInfoCanSkipVerificationResult canSkipVerification(CORINFO_MODULE_HANDLE module)
     BOOL isValidToken(CORINFO_MODULE_HANDLE module, unsigned metaTOK)
     BOOL isValidStringRef(CORINFO_MODULE_HANDLE module, unsigned metaTOK)
+    const char* getStringLiteral(CORINFO_MODULE_HANDLE module, unsigned metaTOK, int* length)
```

Comment:
```suggestion
    LPCWSTR getStringLiteral(CORINFO_MODULE_HANDLE module, unsigned metaTOK, int* length)
```

Reply:
@jkotas it used to be LPCWSTR there but I changed back to `const char*` due to https://github.com/dotnet/runtime/pull/1378#discussion_r370607172 feedback. (because the generator converts LPCWSTR to string). So which one should I use?

Reply:
I would delete this line from ThunkInput.txt and skip any marshaling for LPCWSTR.
```
LPCWSTR,[return: MarshalAs(UnmanagedType.LPWStr)]string,const wchar_t*
```

It should only affect `getJitTimeLogFilename()` that does not do anything (and that should be deleted eventually - we have more general methods to deal with the config settings).